### PR TITLE
* README.md add local development section

### PR DESCRIPTION
**Pull request by mayitbeegh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ token = UserAuth('https://my.cerberus.url', 'username', 'password').get_token()'
 Generally it does NOT make sense to store Lambda secrets in Cerberus for two reasons:
 
 1. Cerberus cannot support the scale that lambdas may need, e.g. thousands of requests per second
-1. Lambdas will not want the extra latency needed to authenticate and read from Cerberus
+2. Lambdas will not want the extra latency needed to authenticate and read from Cerberus
```
Comment (sdford) [inline README.md]: This is a README syntax that converts to the actually numbers for you. That's why you do all ones

Comment (mayitbeegh) [reply README.md]: Updated. Thought I saw 2 1s in preview.

File: README.md [hunk 2]
```diff
@@ -269,6 +270,30 @@ You can run all the unit tests using nosetests. Most of the tests are mocked.
 $ nosetests --verbosity=2 tests/
 ```
 
+
+## Local Development
+
+There are many ways to authenticate with Cerberus and they don't always work on local. Luckily you can easily get a token from the dashboard and export it to your environment variables. Doing so will override the authentication mechanism that's set in place. For example:
```
Comment (sdford) [inline README.md]: I recommend wording this a little more formally. For example:

```The easiest way to locally test the Python client is to first authenticate with Cerberus through the dashboard, then use your dashboard authentication token to make subsequent calls. See examples below.```

File: README.md [hunk 3]
```diff
@@ -269,6 +270,30 @@ You can run all the unit tests using nosetests. Most of the tests are mocked.
...
+from cerberus.client import CerberusClient
+client = CerberusClient('https://dev.cerberus.nikecloud.com') # This will fail on local when it tries to call the metadata endpoint.
+```
+Without changing any code, you can do:
+```bash
+$ export CERBERUS_TOKEN='mytoken'
+```
+```python
+from cerberus.client import CerberusClient
+client = CerberusClient('https://dev.cerberus.nikecloud.com') # On local, the client will pick up the environment variable that was set earlier. When it's deployed to an EC2 instance, it'll automatically switch to authenticating using the metadata endpoint.
```
Comment (sdford) [inline README.md]: This is not necessarily true right?

If the client is deployed on an EC2 instance, but the token environment variable is set, then the environment variable will still be used

File: README.md (hunk 2 above)
Comment (mayitbeegh) [reply README.md]: Copy pasted, also updated one other sentence

File: README.md (hunk 3 above)
Comment (mayitbeegh) [reply README.md]: addressed

**Pull request merged**